src/mappersmith.js

```diff
@@ -1,6 +1,10 @@
 /* global VERSION */
 import ClientBuilder from './client-builder'
 import { assign } from './utils'
+/**
+ * Can be used to test for `instanceof Response`
```

Comment:
I could also use it to create a Response class that extends it, in case I remap the response object in a middleware :)~ I believe there are multiple use cases for it.